src/base/application.cpp

```diff
@@ -266,11 +266,6 @@ void Application::setDefaultDirectories()
 
 #endif
 
-#ifdef DARWIN
-    // add a default data location for Mac OS X
-    inputDirs.push_back("/Applications/Cantera/data");
-#endif
-
```

Comment:
I'm okay with eliminating this as a standard search location, but in that case there are a few docstrings that need to be updated, too (in `global.h` and `application.h`).